pyiceberg/table/__init__.py

```diff
@@ -340,6 +340,34 @@ def set_properties(self, properties: Properties = EMPTY_DICT, **kwargs: Any) ->
         updates = properties or kwargs
         return self._apply((SetPropertiesUpdate(updates=updates),))
 
+    def set_ref_snapshot(
+        self,
+        snapshot_id: int,
+        parent_snapshot_id: Optional[int],
+        ref_name: str,
+        type: str,
+        max_age_ref_ms: Optional[int] = None,
+        max_snapshot_age_ms: Optional[int] = None,
+        min_snapshots_to_keep: Optional[int] = None,
+    ) -> Transaction:
+        """Update a ref to a snapshot.
+
+        Returns:
+            The transaction with the set-snapshot-ref staged
+        """
+        updates = SetSnapshotRefUpdate(
+            snapshot_id=snapshot_id,
+            parent_snapshot_id=parent_snapshot_id,
+            ref_name=ref_name,
```

Comment:
I don't think we are expecting `parent_snapshot_id` in [SetSnapshotRefUpdate](https://github.com/apache/iceberg-python/blob/main/pyiceberg/table/__init__.py#L618-L625) 

```suggestion
            snapshot_id=snapshot_id,
            ref_name=ref_name,
```

Reply:
removed!